file-formats/sito/type/zif_aff_sito_v1.intf.abap

```diff
@@ -0,0 +1,551 @@
+      "! <p class="shorttext">Entity Set</p>
+      "! Name
+      entity_set TYPE c LENGTH 40,
+    END OF ty_sit2_obj_vh_s_p,
+
+    "! <p class="shorttext">Value Help Service Providers</p>
+    "! Property of value help service for situation object
+    ty_sit2_obj_vh_s_p_list TYPE STANDARD TABLE OF ty_sit2_obj_vh_s_p WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object text
+    BEGIN OF ty_sit2_object_t,
+      "! <p class="shorttext">Name</p>
+      "! Situation object name
+      name        TYPE c LENGTH 40,
+      "! <p class="shorttext">Description</p>
+      "! Description
+      description TYPE c LENGTH 255,
+    END OF ty_sit2_object_t,
+
+    "! <p class="shorttext">Texts</p>
+    "! Situation object text
+    ty_sit2_object_t_list TYPE STANDARD TABLE OF ty_sit2_object_t WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Structures</p>
+    "! Situation object structure
+    BEGIN OF ty_sit2_obj_struc,
+      "! <p class="shorttext">Reusable</p>
+      "! Object reusable
+      reusable             TYPE abap_bool,
+      "! <p class="shorttext">Type</p>
+      "! Situation object structure type
+      "! $required
+      type                 TYPE ty_sit2_do_obj_struc_type,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope                TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">Name</p>
+      "!
+      name                 TYPE c LENGTH 30,
+      "! <p class="shorttext">SAP Object Node Type</p>
+      "! Sap object node type
+      sap_object_node_type TYPE c LENGTH 30,
+      "! <p class="shorttext">ID</p>
+      "! Situation object structure id
+      id                   TYPE c LENGTH 32,
+      texts                TYPE ty_sit2_obj_struc_t_list,
+      semantic_keys        TYPE ty_sit2_obj_str_sk_list,
+      value_help_mappings  TYPE ty_sit2_os_vh_map_list,
+      services             TYPE ty_sit2_os_vh_map_s_list,
+    END OF ty_sit2_obj_struc,
+
+    "! <p class="shorttext">Structures</p>
+    "! Situation object structure
+    ty_sit2_obj_struc_list TYPE STANDARD TABLE OF ty_sit2_obj_struc WITH DEFAULT KEY,
+
+    "! <p class="shorttext">Events</p>
+    "! Situation object events
+    BEGIN OF ty_sit2_obj_events,
+      "! <p class="shorttext">Reusable</p>
+      "! Object reusable
+      reusable               TYPE abap_bool,
+      "! <p class="shorttext">Category</p>
+      "! Object type category
+      "! $required
+      category               TYPE ty_swfeclstyp,
+      "! <p class="shorttext">Scope</p>
+      "! Source
+      "! $required
+      scope                  TYPE ty_sit2_do_scope,
+      "! <p class="shorttext">ID</p>
+      "! Situation event id
+      id                     TYPE c LENGTH 32,
+      "! <p class="shorttext">Object</p>
+      "! Object type
+      object                 TYPE c LENGTH 32,
+      "! <p class="shorttext">Type</p>
+      "! Object type
+      type                   TYPE c LENGTH 32,
```

Comment:
Not sure what the difference between `object` and `type` is. But it is a bit misleading that they have the same description. Seems like they are the same